app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<h2>Log in</h2>
+
+<div class="auth">
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email" %>
+    </div>
+
+    <div class="field">
+      <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div>
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions button">
+      <%= f.submit "Log in" %>
+    </div>
+  <% end %>
+
+</div>
+
+<%= render "devise/shared/links" %>
+
+<%= link_to "Back", :back, class: "button" %>
```

Comment:
- [ ] Kindly center the log in form too, based on earlier comment on the sign up page.

  ![image](https://github.com/Arch-Noize/rorblog/assets/81468675/43fe857e-d599-41eb-904c-bea62b984dfa)